c-api/dotnet/src/Picky/Error.cs

```diff
@@ -0,0 +1,39 @@
+namespace Devolutions.Picky;
+
+public class ErrorException : System.Exception
+{
+    public ErrorException(String message) : base(message) { }
```

Comment:
nit: It's more idiomatic to use the `string` alias than `String`. You can do what you like best, however! But be advised that StyleCop won't like that if you start using it (it only wants you to use `String` when calling static functions on the `String` class...)

Reply:
Thank you! That's my first code in C# so I'm not really aware of the idioms, I'll definitely follow you on that